locale/core.yml

```diff
@@ -238,8 +238,11 @@ core:
 
     # These translations are used in the Edit User modal dialog (admin function).
     edit_user:
+      activate: Activate User
       email_label: => core.ref.email
```

Comment:
That's a tricky point. The headings were originally called `_label` because it seemed like each corresponds to only one setting. But when you compare them with "Groups" it looks like they should be considered headings.

Anyway, that's my take on it: they're in boldface, so let's call them headings. (That boldface is what's used for headings elsewhere in the UI.) But let's see what @tobscure thinks.